backend/code_review_backend/issues/api.py

```diff
@@ -152,6 +152,9 @@ class IssueCheckStats(generics.ListAPIView):
     queryset = (
         Issue.objects.values("analyzer", "check", "diff__revision__repository__slug")
         .annotate(total=Count("id"))
+        .annotate(
+            publishable=Count("id", filter=Q(in_patch=True) & Q(new_for_revision=True))
```

Comment:
Shouldn't it be OR? If the issue is in the patch, it is publishable. If the issue is not in the patch, but it is new, it is publishable.

Reply:
That would make all issues detected in the first patch publishable; to be consistent with current usage, we should only report `in_patch`.
I added the `new_for_revision` to avoid re-publishing exact duplicate in the revision once we switch the bot

Reply:
Ah, I wrongly remembered new_for_revision was False for all issues in the first diff of a revision.

Anyway, when we will start publishing issues as Phabricator lint issues, Phabricator will not show issues from a previous diff on the current diff, so the `new_for_revision` check will become harmful. It might be harmful now too, as developers might assume an issue doesn't apply anymore if the comment for the current push doesn't contain that issue. Maybe we should just have publishable == in_patch for now.

Reply:
I made `publishable == in_patch` for now.